### ISSUE TITLE
Where is the checkpoint and training set ?

### ISSUE THREAD

**Issue by kkennethwu** [arthurchen0518/DirectionNet#2]
Hello! I'm trying to use your code for some application. In your instruction for training rotation, what are the <path_to_checkpoints_and_logs> and <path_to_training_set>? If I use "MatterportA_checkpoints", where are they?

Thanks a lot !!

**Comment by arthurchen0518** [comment]
<path_to_checkpoints_and_logs> is the directory you create to store the TensorFlow checkpoints and event files.  <path_to_training_set> is the directory to the training data. "MatterportA_checkpoints" contains the TensorFlow checkpoints that we saved when we were training the model on the MatterportA dataset in the paper. Right now, due to the large size of the dataset, we haven't been able to release it. Our datasets are generated using `generate_random_views `in dataset.py from [Matterport3D](https://niessner.github.io/Matterport/) given the panorama images and camera labels. You can also use your own stereo dataset to train the model as long as the stereo images have the same intrinsic.

**Comment by kkennethwu** [comment]
Oh ! I got it. Thanks for your help !!!
